.github/workflows/sync_docs.yaml

```
@@ -0,0 +1,19 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 50 00 * * *
+
+jobs:
+  sync-docs:
+    name: Sync docs from Discourse
+    uses: canonical/data-platform-workflows/.github/workflows/_sync_docs.yaml@v14.0.0
```

```suggestion
    uses: canonical/data-platform-workflows/.github/workflows/_sync_docs.yaml@main
```
This workflow is still going through some changes, so to avoid having to update this every time, I would use the `main` tag for now.